app/routes/videos/$id.tsx

```diff
@@ -641,7 +641,7 @@ function HighlightText({
   return (
     <>
       {t1}
-      <span className="underline">{t2}</span>
+      <span className="border-b border-colorTextSecondary pb-0.2">{t2}</span>
```

Comment:
padding-bottom seems no-op when multiline. Well I didn't even expect for padding to work at all for inline, so it's fine for now.

Reply:
Not sure what I was looking at, but it looks fine actually.